src/pages/Contact.tsx

```
@@ -1,15 +1,71 @@
+import LoadingBtn from "../components/LoadingBtn"
 
 
 const Contact = () => {
+
+  const [firstName,setFirstName] = useState("")
+  const [lastName,setLastName] = useState("")
+  const [email,setEmail] = useState("")
+  const [subject,setSubject] = useState("")
+  const [message,setMessage] = useState("")
+
+const [contact,{isLoading}] = useContactMutation() as any;
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)